tgfilestream/config.py

```diff
@@ -56,3 +56,6 @@
 except ValueError:
     print("Please make sure the CONNECTION_LIMIT environment variable is an integer")
     sys.exit(1)
+
+
+start_message = os.environ.get("TG_START_MESG", "/start")
```

Comment:
`/start` doesn't really make sense as a reply

```suggestion
start_message = os.environ.get("TG_START_MESG", "Send an image or file to get a link to download it")
```